plugins/inputs/csgo/README.md

```diff
@@ -0,0 +1,33 @@
+# CSGO Input Plugin
+
+The `csgo` plugin gather metrics from CSGO servers.
+
+#### Configuration
+```toml
+[[inputs.csgo]]
+  servers = [
+    ["ip1:port1", "rcon_password1"],
+    ["ip2:port2", "rcon_password2"],
+  ]
+```
```

Comment:
`SampleConfig must match the readme, but not include the plugin name.` -> https://github.com/influxdata/telegraf/wiki/Review

Please, copy here content of `sampleConfig` from `csgo.go`